src/vos/vos_dtx.c

```diff
@@ -41,6 +41,12 @@
 #define DTX_ACT_BLOB_MAGIC	0x14130a2b
 #define DTX_CMT_BLOB_MAGIC	0x2502191c
 
+static inline bool
+umoff_is_null(umem_off_t umoff)
```

Comment:
seems a little confuse compared to existing macro UMOFF_IS_NULL, can it directly use UMOFF_IS_NULL?

Reply:
UMOFF_IS_NULL() will hide um_flags, such as DTX_UMOFF_ABORTED, means that the "UMOFF_IS_NULL(DTX_UMOFF_ABORTED)" will be true, that is different from umoff_is_null().